src/v2/Apps/Consign/Components/SellArtDifferently.tsx

```diff
@@ -1,39 +1,48 @@
 import React from "react"
-import { Box, EditIcon, Flex, Separator, Text } from "@artsy/palette"
+import { Box, EditIcon, Flex, Separator, Text, color } from "@artsy/palette"
+import { SectionContainer } from "v2/Apps/Artist/Routes/Consign/Components/SectionContainer"
+import { Media } from "v2/Utils/Responsive"
 
 export const SellArtDifferently: React.FC = () => {
   return (
-    <Box>
+    <SectionContainer>
       <Box>
-        <Text variant="title">Selling Art Differently</Text>
-        <Separator />
+        <Media greaterThanOrEqual="md">
+          <Text textAlign={["center", "left"]} mb={2} variant={"title"}>
+            Selling Art Differently
+          </Text>
+        </Media>
+        <Media lessThan="md">
+          <Text textAlign={["center", "left"]} mb={2} variant={"largeTitle"}>
+            Selling Art Differently
+          </Text>
+        </Media>
```

Comment:
This might change per our convo in #design-system but will leave it to you on whether we should update or merge as is!